src/main/cliapp/src/App.js

```diff
@@ -131,6 +131,7 @@ const App = () => {
         {
             label: 'Ontologies', icon: 'pi pi-fw pi-sitemap',
             items: [
+                { label: 'CHEBI Ontology', icon: 'pi pi-fw pi-home', to: '/ontology/chebi' },
```

Comment:
For consistency, should be:
`{ label: 'Chemical Entities of Biological Interest (ChEBI)', icon: 'pi pi-fw pi-home', to: '/ontology/chebi' },`